ntb_hw_switchtec.c

```diff
@@ -1320,34 +1331,61 @@ static int switchtec_ntb_init_crosslink(struct switchtec_ntb *sndev)
 	if (rc)
 		return rc;
 
-	rc = crosslink_setup_mws(sndev, dbmsg_lut_idx, &bar_addrs[1],
-				 bar_cnt - 1);
-	if (rc)
+	sndev->mmio_xlink_dbmsg_win = pci_iomap_range(sndev->stdev->pdev, bar,
+						      dbmsg_lut_idx * LUT_SIZE,
+						      LUT_SIZE);
+	if (!sndev->mmio_xlink_dbmsg_win) {
+		rc = -ENOMEM;
 		return rc;
+	}
 
-	rc = crosslink_setup_req_ids(sndev, sndev->mmio_peer_ctrl);
+	sndev->mmio_peer_dbmsg = sndev->mmio_xlink_dbmsg_win + offset;
+	sndev->nr_rsvd_luts++;
+
+
```

Comment:
Nit: extra blank line

Reply:
Yeah, it's a bit. I looked again, it seems to me all the changes are related to the introduction of the new LUT window, splitting them might break the consistency. I'd like to keep it as is. 

Thanks.